testpilot/frontend/static-src/app/views/header-view.js

```diff
@@ -74,6 +74,11 @@ export default BaseView.extend({
       credentials: 'same-origin',
       headers: { 'X-CSRFTOKEN': app.me.csrfToken }
     }).then(() => {
+      app.sendToGA('event', {
```

Comment:
I'm going to remove the extra logout call here since it is only used in `settings-menu-view.js`